lib/exactor/common.ex

```diff
@@ -0,0 +1,22 @@
+defmodule ExActor.Common do
+  @moduledoc false
+
+  # This module adds common functionality to all ExActor behaviours.
+  defmacro __using__(_opts) do
+    quote do
+      @doc """
+      By default, the `server` argument given
+      to the different interface functions, is expected to be a process identifier,
+      unless overridden by the `export:` option.
+
+      But by providing a custom implementation of `server_pid/1`, you can map an identifier
+      to a PID by some other means.
+      """
+      def server_pid(server_reference) do
```

Comment:
I wonder if we should make this public. By doing it, we're by default adding extra function to the module's interface, and I'm not sure I like that. Preferably, this would be a `defp`, though I'm not sure if `defoverridable` would work then. If it doesn't, then we could make it `@doc false` so it's not included in the generated docs, and marked as an internal detail.